src/acl-control.js

```diff
@@ -612,14 +612,19 @@ UI.aclControl.ACLControlBox5 = function (subject, dom, noun, kb, callback) {
           var q = str.indexOf('//')
           var targetDocDir = ((q >= 0 && p < q + 2) || p < 0) ? null : str.slice(0, p + 1)
 
-          if (targetDocDir) {
+          // @@ TODO: The methods used for targetIsStorage are HACKs - it should not be relied upon, and work is
+          // @@ underway to standardize a behavior that does not rely upon this hack
```

Comment:
Ideally, you'd also include a link here to where one can check the status of the work that is underway, so that engineers coming across this in the future can tell whether the hack can be removed. (And for bonus points: a note about what the risks of the current hack are.)